util/dvsim/dvsim.py

```diff
@@ -483,7 +485,8 @@ def parse_args():
     rung.add_argument("--run-timeout-mins",
                       type=int,
                       help=('Wall-clock timeout for runs in minutes: if '
```

Comment:
Should we default this to 60 as well?

Reply:
I didn't revise `run_timeout_mins` here yet. Maybe I can try change it too.